### ISSUE TITLE
/annotations/instances_valminusminival2018.json file missing

### ISSUE THREAD

**Issue by BennoStaub** [waspinator/pycococreator#3]
Trying to train Mask R CNN, I get the error that above file is missing. 
Did you try to use your created dataset for a model like Mask R CNN? Maybe there are more files missing. Eager to hear from you and thx.

**Comment by hiankun** [comment]
I am not sure whether my problem is the same one in this topic.

When I tried to run `visualize_coco.ipynb`, it said there's no file named `instances_shape_train2018.json`. Then I found it via your article in which I downloaded `shapes_train_dataset` to replace the one given by the Jupyter notebook example.

Maybe it's a good idea to update the `examples/shapes/train` content with corresponding json file, or indicate the download link in README.md?

**Comment by gaqiness** [comment]
It is not in GitHub, I think you can download from
 https://patrickwasp.com/wp-content/uploads/2018/04/shapes_train_dataset.zip

**Comment by austinmw** [comment]
You need to comment out the line in coco.py where it tries to load that (it's trying to append an additional set of data for training). Also change the import minival line to just import val.
